addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/json/AirQualityJsonTime.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal.json;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * The {@link AirQualityJsonTime} is responsible for storing
+ * the "time" node from the waqi.org JSON response
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityJsonTime {
+
+    private String s;
+    private String tz;
+
+    private final static SimpleDateFormat SDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * Get Timezone in following format: "+0100"
+     *
+     * @return {String}
+     */
+    public String getTz() {
+        return tz.replace(":", "");
```

Comment:
I was checking that and there is no nice way to convert time representation given by aqicn to `java.util.Date` or `Calendar` directly. Sadly there is none. With given data
`{v: int, s: string, tz:"01:00"}` we could convert unix timestamp coded in `v` field of json to java timestamp (coded in millis), but then we have to do dance with timezone. This means that only custom Gson `TypeAdapter` will cover that. In this way we would close time related handling in separate type, but since there is no other uses of this structure benefits are minor.  Without joda time or java 8 date-time api it will always be counter intuitive.
One thing we could do is change of simple date fromat to `yyyy-MM-dd HH:mm:ssX` to parse concatenated `s+ tz` fields easily. `X` allows to specify time zone offset in format returned by service.